### ISSUE TITLE
Really slow download (just rn)

### ISSUE THREAD

**Issue by proall** [ByteDream/crunchyroll-go#12]
![Screenshot_1](https://user-images.githubusercontent.com/36814374/145476963-fa864001-bcd4-47c1-9824-e6d746d67ea0.png)

This, took me 5 min.
I restarted my pc + network.
What can i do?

**Comment by ByteDream** [comment]
What did you try to download?
For me the downloading works fine

**Comment by proall** [comment]
Yeah, even the in the website i can't play a single episode without bags... :/

**Comment by YoSiJo** [comment]
I can't confirm the problem either. I therefore think that it is not a problem of the software here and the issue can be closed.
